### PR TITLE
Gus blog post on UX - Part 3 and relevant images

### PR DESCRIPTION
**Pull request by gusferguson**
Part 3 of my blog post plus the images in it.

Needs proof reading and general sense-making check.

### CODE REVIEW

File: _posts/2015-06-26-ux-part-3.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+*Version 5.0.6 release.*
+
+##Second Round
+
+In the second round we decided to take the opportunity to rationalise and improve the controls and layout of the rendering settings panel. Various layouts were scoped out so the team could get a proper feel for them. You can see the playing of musical toolbars here.
+
+<img src="/images/state-6.png" alt="Screenshots of mock-ups showing various toolbar placement options in OMERO.insight." style="width: 600px;"/>
+
+*Different toolbar arrangements visulaised.*
```
Comment (hflynn) [inline _posts/2015-06-26-ux-part-3.md]: visulaised -> visualised

File: _posts/2015-06-26-ux-part-3.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+And this will hopefully be implemented in the next round of UI improvements.
+
+Another important advantage with this workflow is that with all the scoping done well in advance of actual implementation, it helps to avoid conflict with other changes in the pipeline. So now if another change is muted that affects the Preview pane, Full Viewer or rendering settings, it is relatively easy to make sure that it fits in with these changes.
+
+##Release
+
+The UX team’s job doesn’t end with code implementation, as we don’t just abandon the user at this stage. With the OME’s continuous integration development system, the code, the developer documentation and the User Help are all updated and held in repositories ready for release.
+
+So when all the testing is complete and everyone’s happy, Sébastien presses The Big Red Button and the version release of all the elements is done at once.
```
Comment (hflynn) [inline _posts/2015-06-26-ux-part-3.md]: This slightly makes Seb sound like the project gatekeeper! Maybe better to just say 'The Big Red Button is pressed'

File: _posts/2015-06-26-ux-part-3.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+<img src="/images/iterations.png" alt="Image showing the number of files in a scoping folder." style="width: 600px;"/>
+
+*The extent of the scoping work and iterations.*
+
+Once we have implemented something we do try and remember to record screenshots of the released version and add them to the team Google Drive as a record of the final implementation.
+
+##In Summary
+
+So while I admit user-centred design is not always the easiest road to travel in small, expert system developments, it still gives good results. As we have pressed on with this approach, we have seen what were a pretty high level of user complaints, and sometimes even downright hostility, slowly morph into much more positive feedback. From: “why can’t I do this, your software is useless” to: “it would be really cool if I could also do that”.
```
Comment (hflynn) [inline _posts/2015-06-26-ux-part-3.md]: I'm not sure if Jason will be happy with being this negative about our earlier incarnations, could we just say, feedback is much more positive (although I think it's fine to leave in the example comment, just not the "pretty high level od user complaints, and sometimes even downright hostility" bit) ?

File: _posts/2015-06-26-ux-part-3.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+Our key aim is to try and identify and sort out issues and problems on paper during the design process before committing our very limited resources of developer time to coding. Done properly, a few rounds of wrangling by the team at the design stage, can save weeks of pain and suffering at the implementation and testing stages. 
+
+Visualisation of the problem and potential solutions is really the key to all of this, be it sketches, mock-ups or paper prototypes. The devil is in the detail and often the only way to hunt it out is to put it all down in black and white and make sure it works before the coding starts.
+
+And you know when you have succeeded when you take the new features back to the user and they and turn around and say - “you see it was really simple – why did it take you so long to do it that way”. They don’t see all the sweat, the blood on the walls, the hundreds of rejected solutions and the hammering out of every little detail that went into that final implementation.
```
Comment (hflynn) [inline _posts/2015-06-26-ux-part-3.md]: few too many "and"s here - "and they turn around and say"

**Comment by hflynn** [comment]
Just a few nitpicks here.

**Comment by hflynn** [comment]
Looks good to me now!

File: _posts/2015-06-26-ux-part-3.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: The OME UX Team. Part 3 - Mock-ups and Implementation in Stages
+categories: user-experience
+---
+*By Gus Ferguson*
+
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: Same enumeration than the one at the end of post 2.
Is that the intension?

File: _posts/2015-06-26-ux-part-3.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+Along with the changes that had been requested, the UX team also wanted to:
+
+- make it easier to see the rendering settings saved by other users, which was previously pretty well hidden in the UI, 
+- improve the layout and availability of the rendering settings controls and 
+- at the same time bring the UIs for this functionality in both the clients into line.
+
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: if they are in a shared group.

File: _posts/2015-06-26-ux-part-3.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+And we realised we also had to scope all the relevant contextual menus.
+
+<img src="/images/state-4.png" alt="Mock-up of OMERO.insight Centre and Preview panes showing contextual menus." style="width: 600px;"/>
+
+*Contextual menus.*
+
+##First Round
+
+At that particular stage in the development cycle, the team was close to a scheduled release, so there was not a lot of time to fit in this new functionality. Therefore the implementation for the version 5.0.6 release was limited to exposing the saved user settings as thumbnails, but in OMERO.insight only.
+
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: Maybe a comment: ````even developers can be reasonable sometimes!!!```

File: _posts/2015-06-26-ux-part-3.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+*Version 5.0.6 release.*
+
+##Second Round
+
+In the second round we decided to take the opportunity to rationalise and improve the controls and layout of the rendering settings panel. Various layouts were scoped out so the team could get a proper feel for them. You can see the playing of musical toolbars here.
+
+<img src="/images/state-6.png" alt="Screenshots of mock-ups showing various toolbar placement options in OMERO.insight." style="width: 600px;"/>
+
+*Different toolbar arrangements visualised.*
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: you use visualize earlier?

File: _posts/2015-06-26-ux-part-3.md (hunk 6 above)
Comment (gusferguson) [inline _posts/2015-06-26-ux-part-3.md]: Yes.

Comment (gusferguson) [inline _posts/2015-06-26-ux-part-3.md]: Happy to remove it it looks odd.

File: _posts/2015-06-26-ux-part-3.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+---
+*By Gus Ferguson*
+
+Along with the changes that had been requested, the UX team also wanted to:
+
+- make it easier to see the rendering settings saved by other users, which was previously pretty well hidden in the UI, 
+- improve the layout and availability of the rendering settings controls and 
+- at the same time bring the UIs for this functionality in both the clients into line.
+
+Mock-ups of the proposed changes to the UI are done using screenshots and mocked-up elements in Omnigraffle, allowing the team to visualize what the changes will look like and discuss improvements. The mock-ups follow the actions of the user carrying out tasks using the proposed UI allowing any issues or problems to become readily apparent. They also ensure that all steps in workflows can be carried out using the new interface and all user requirements are met.
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: visualise

File: _posts/2015-06-26-ux-part-3.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+<img src="/images/release.png" alt="Image showing the release download page, developer documentation and User Help." style="width: 600px;"/>
+
+*Similtaneous release of all elements.*
+
+The benefits to the team of this coordination using the continuous integration and release system have been huge, making the testing and review of updates as painless as humanly possible.
+
+##Iterations Count
+
+This is the Scoping folder on my Mac - 32 separate scoping exercise so far. Some, like  number 4 - Tagging, have up to nine versions of iteration.
+
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: I am not sure that give the good message. People might ask how do you share?

File: _posts/2015-06-26-ux-part-3.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+<img src="/images/iterations.png" alt="Image showing the number of files in a scoping folder." style="width: 600px;"/>
+
+*The extent of the scoping work and iterations.*
+
+Once we have implemented something we do try and remember to record screenshots of the released version and add them to the team Google Drive as a record of the final implementation.
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: same here, ```we do try``` for me makes things that we often fail

File: _posts/2015-06-26-ux-part-3.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+
+##In Summary
+
+So while I admit user-centred design is not always the easiest road to travel in small, expert system developments, it still gives good results. As we have pressed on with this approach, we have seen much more positive feedback, with a greater number of users engaging with, and participating in, the development process. An indication of the effectiveness of the approach can be seen by the feedback from users being much more about how we could add to or enhance functionality rather than being mainly about issues with the behaviour, or mis-behaviour, of the user interface.
+
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: A quote/note from Paris 2015. ```No real user complains this year``` (at least not in the form your tools suck!!)

File: _posts/2015-06-26-ux-part-3.md [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+Our key aim is to try and identify and sort out issues and problems on paper during the design process before committing our very limited resources of developer time to coding. Done properly, a few rounds of wrangling by the team at the design stage, can save weeks of pain and suffering at the implementation and testing stages.
```
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: remove very.

File: _posts/2015-06-26-ux-part-3.md (hunk 7 above)
Comment (gusferguson) [inline _posts/2015-06-26-ux-part-3.md]: A bit of developer paranoia creeping in here?? :-)

Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: More than they can see the light ...sometimes!!!!

File: _posts/2015-06-26-ux-part-3.md (hunk 6 above)
Comment (jburel) [inline _posts/2015-06-26-ux-part-3.md]: It does not look odd just wanted to double-check.

**Comment by gusferguson** [comment]
@hflynn @jburel - OK I think I have done everything for this part.

**Comment by jburel** [comment]
```Once we have implemented something``` => I will replace it 
Once a scope feature has been implemented, we record ....

**Comment by gusferguson** [comment]
@jburel - done.

**Pull request closed without merging**